sql/engines/__init__.py

```diff
@@ -17,18 +18,36 @@ def __init__(self, instance=None):
             self.password = instance.password
             self.db_name = instance.db_name
 
+            # 判断如果配置了隧道则连接隧道，只测试了MySQL
+            if self.instance.tunnel != None:
```

Comment:
判断none 的都不需要这么判断, 直接if 就行
```suggestion
            if self.instance.tunnel:
```